### ISSUE TITLE
Proposal to reorder elements in the New Charsheet sidebar

### ISSUE THREAD

**Issue by cswendrowski** [foundryvtt/pf2e#509]
In GitLab by @apoapostolov on Jun 3, 2020, 13:54

The current state as of 1.4.2. It makes sense to order elements by how useful they are in normal encounters. Currently Perception and Initiative are first which is exactly right and usually Initiative is based on Perception, they should be together and one next to the other. But Class DC is far less important than saves, and few mechanics use it. I propose that Class DC is moved below Saves, not above them. 

![image](http://gitlab.com/hooking/foundry-vtt---pathfinder-2e/uploads/ff4f95bb05e3a4b5484616de054f7454/image.png)
